gpii/node_modules/solutionsRegistry/test/SolutionsRegistryFileTests.js

```diff
@@ -65,18 +66,77 @@ gpii.tests.universal.solutionsRegistry.srFiles.checkSingleSolution = function (g
 
 // Ensure that all "capabilities" refer to a generic preference setting.
 gpii.tests.universal.solutionsRegistry.srFiles.checkCapabilities = function (filename, solutionKey, solutionDef, genericTerms) {
+    var missingTermsCount = 0;
     if (solutionDef.capabilities && solutionDef.capabilities.length) {
         fluid.each(solutionDef.capabilities, function (capabilityKey) {
             var unescapedKey = capabilityKey.replace(/\\/g, "");
             var termDef  = genericTerms[unescapedKey];
-            jqUnit.assertTrue("The capability '" + capabilityKey + "' specified in solution '" + solutionKey + "' should exist.", termDef !== undefined);
+            if (termDef === undefined) {
+                missingTermsCount++;
+                fluid.log("The capability '" + capabilityKey + "' specified in solution '" + solutionKey + "' does not exist.");
+            }
         });
+
+        jqUnit.assertEquals("There should be no missing capabilities in solution '" + solutionKey + "'.", 0, missingTermsCount);
     }
     else {
         jqUnit.assert("Entry '" + solutionKey + "' has no capabilities, which is fine.");
     }
 };
 
+// Ensure that all transforms involving common terms refer to common terms that actually exist.
+gpii.tests.universal.solutionsRegistry.srFiles.checkTransforms = function (filename, solutionKey, solutionDef, genericTerms) {
+    fluid.each(solutionDef.settingsHandlers, function (settingsHandlerDef, settingsHandlerKey) {
+        var allTransformTerms = [];
+
+        if (settingsHandlerDef.capabilitiesTransformations) {
+            var forwardTransformTerms = gpii.tests.universal.solutionsRegistry.srFiles.extractCommonTerms(settingsHandlerDef.capabilitiesTransformations);
+            if (forwardTransformTerms.length) {
+                allTransformTerms = allTransformTerms.concat(forwardTransformTerms);
+            }
+        }
+
+        if (settingsHandlerDef.inverseCapabilitiesTransformations) {
+            var inverseTransformTerms = gpii.tests.universal.solutionsRegistry.srFiles.extractCommonTerms(settingsHandlerDef.capabilitiesTransformations);
```

Comment:
This incorrectly extracted terms from the forward transforms again rather than actually inspecting the inverse transforms.  Fixed.